profile/Overview.md

```diff
@@ -0,0 +1,66 @@
+
+# Confidential Containers
+We are interested in integrating existing [Trusted Execution Environments](https://en.wikipedia.org/wiki/Trusted_execution_environment) (TEE) 
+infrastructure support and technologies with the cloud native world. Our focus is to place a kubernetes pod into a TEE. 
+
+## Why?
+Security has long been a significant concern with data encryption at rest and in flight assumed to be a key part of any offering.
+Trusted Execution Environments look to address the data in use security concern.
+
+Cloud Computing adoption continues to accelerate whether it be Public, Private or increasingly common a Hybrid approach
+and with it the trust boundaries change. Consideration of Insider Threats needs to now consider the cloud provider,
+infrastructure provider, managed service provider.
+
+Certain Industries are heavily focused on compliance to standards, governments too are concerned both [collectively](https://www.un.org/counterterrorism/cybersecurity)
+and [individually](https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/).
+The standards expected to protect software solutions continue to evolve towards a concept of Confidential Computing
+
+Confidential Containers look to address the growing conern and needs of these three areas combining and growing in the future.
+
+## How?
+### Trusted Execution Environments
+![TEE Protects an application](../images/ApplicationTEEProtection.png)
+
```

Comment:
let's be really clear about the wide array of technologies that we support.

Reply:
Fixed in new pull request her -> https://github.com/confidential-containers/documentation/pull/3